addon/schema-manager.js

```diff
@@ -12,9 +12,10 @@ export class SchemaManager {
   }
 
   computeBaseModelName(projectionModelName) {
-    if (typeof this.schema.computeBaseModelName !== 'function') {
-      return null;
+    if (!this.schema || typeof this.schema.computeBaseModelName !== 'function') {
```

Comment:
None of the other functions checks whether `this.schema` is set, which is not a terrible thing, but makes this function stand out... kind of.

Reply:
do not silently tolerate `undefined` schema

Reply:
👍